### PR TITLE
feat: add --config flag to allow different config files to be passed

### PR DESCRIPTION
**Pull request by redhatrises**
This PR allows different config files to be passed to make submitting different configurations easier.

### CODE REVIEW

**Comment by redhatrises** [comment]
@acornett21 bump on this PR

File: cmd/preflight/cmd/root.go [hunk 1]
```diff
@@ -35,6 +35,8 @@ func rootCmd() *cobra.Command {
 	}
 
 	viper := viper.Instance()
+	rootCmd.PersistentFlags().String("config", "", "config file (default is config.yaml)")
```
Comment (bcrochet) [inline cmd/preflight/cmd/root.go:38]: Please add the env equivalent to the docstring.

Comment (redhatrises) [reply cmd/preflight/cmd/root.go]: @acornett21 done!

**Review by bcrochet — APPROVED** [review]
/lgtm

**Review by acornett21 — APPROVED** [review]

**Pull request merged**